docs/plugins/text-to-speech-announcer.md

```diff
@@ -47,9 +47,65 @@ Blits.Component('MyComponent', {
 })
 ```
 
+### Execute logic when done
+
+The `speak()`-function returns a promise that resolves when it's done speaking, or when it's interupted or errors out. The reason of the `speak()` method resolving is passed as an argument into the resolve function.
+
+```js
+this.$announcer.speak('Hello').then((reason) => console.log('Speaking hello finished', reason))
+```
+
+### Announcement queue
+
+The `speak()`-method places the messages in an announcement queue and then starts processing that queue. That means that when the queue is empty, it will instantly announce the messages passed. If there are other messages in the queue, the messages will be spoken when all previous messages are announced.
+
+Optionally you can set the politeness of a message to `assertive` and pass it a the second argument of the `speak()`-method.
+
+```js
+this.$announcer.speak('Very important message that should get priority', 'assertive')
+```
+
+This will place the message at the beginning of the queue, making in the first message to be announced after the current message (if any) has finished.
+
+#### Clearing and interupting
+
+Since each message added into the queue may take a bit of time to actually be announced, it's possible that the user has navigated elsewhere in the mean time, making the queued up messages not relevant.
+
+The announcer plugin allows you to clear the entire queue by calling `this.$announcer.clear()`. Additionally if you want to interup the messages being currently spoken, the announcer plugin offers the `this.$announcer.stop()`-method.
+
+#### Canceling individual messages
+
+In some cases you may not want to clear the entire queue, but instead cancel out a single message.
+
+Imagine an App with a row of tiles, it's possible that before the title of the role is being spoken out, the user alrady navigates through the tiles within the row. Traditionally you'd use the focus event to speak out info about each tile (i.e. adding tot the queue). You don't want all previously focused tiles to still be announced, but tou still would want the category of the row to be announced for example, so clearing out the entire queue is too much.
```

Comment:
alrady -> already.
last sentence:
You don't want all previously focused tiles to still be announced, but tou still would want the category of the row to be announced for example, so clearing out the entire queue is too much.
to:
You don't want all previously focused tiles to still be announced, but would still want the category of the row to be announced, making clearing the queue not required.